apps/omg_db/lib/omg_db/models/payment_exit_info.ex

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.DB.Models.PaymentExitInfo do
```

Comment:
`Models` makes me thinks about `DTOs`, schemata and things from SQL world...
also the module doc suggest it is Payment Exit Info model  - but in fact it's a method to obtains (query) such values.

Reply:
yeah, though not really `DTO`, but I was thinking [DAO](https://en.wikipedia.org/wiki/Data_access_object) which is in practice having high change to be DTO as well from experience.

Your comment is totally valid. This PR failed to introduce all the things that make it similar to DAO which I later realized that some modules under `ExitProcessor` is acting quite similar to it already. eg. [ExitInfo](https://github.com/omisego/elixir-omg/blob/e8d486d2cd19808d8a5e56d4a07412645813441c/apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex#L15)

Probably we can rename this to be `Queries.PaymentExitInfo` for now (?) and when we refactor the part touching exit processor we can try to combine the existing `ExitInfo` with the `Query` to be a single module when I understand how that is used more?